servant-activeresource.cabal

```diff
@@ -47,7 +47,7 @@ common deps
     , containers        ^>=0.6     || ^>=0.7
     , servant           >=0.19     && <0.21
     , servant-server    >=0.19     && <0.21
-    , template-haskell  >=2.16     && <2.22
+    , template-haskell  >=2.16     && <2.23
```

Comment:
`template-haskell` is coupled to GHC versions so I don't think we should raise this until we support 9.10.

Reply:
K thanks